backend/apps/volontulo/serializers.py

```diff
@@ -291,3 +301,16 @@ class ContactSerializer(serializers.Serializer):
     )
     message = CharField(required=True, min_length=10, max_length=2000)
     phone_no = CharField(max_length=20)
+
+
+class PasswordChangeSerializer(serializers.Serializer):
+    """Serializer for password reset when user is logged in."""
+    password_old = PasswordField(required=True)
+    password_new = PasswordField(required=True)
+
+    def validate_password_old(self, value):
+        """Checks that password_old matches user's password."""
+        value = super().validate(value)
```

Comment:
that's not the same method in parent class and I believe it will be called anyhow.